app/assets/javascripts/host_edit_interfaces.js

```diff
@@ -391,7 +391,9 @@ $(document).on('change', '.virtual', function() {
 function construct_host_name() {
   var host_name_el = $('#host_name')
   var host_name = host_name_el.val();
-  if (host_name_el.data('appendDomainNameForHosts') === false) {
+  if (host_name_el.data('appendDomainNameForHosts') === false ||
+      host_name_el.data('managed') === false
```

Comment:
This was removed in https://github.com/theforeman/foreman/commit/f13142e9e5e8cc9561e9934181fb4a0faf7567a6. Probably a bad rebase?

Reply:
Yeah.